client/src/Questions/QuestionForm.js

```diff
@@ -9,9 +9,12 @@ import ValidationError from '../ValidationError';
 
 function QuestionForm() {
   const { id } = useParams();
+  const query = new URLSearchParams(useLocation().search);
   const history = useHistory();
   const [error, setError] = useState(null);
+  const [questionnaires, setQuestionnaires] = useState([]);
```

Comment:
Here we set up new state variable/change functions to store the list of questionnaires